parsl/tests/site_tests/conda_setup.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+echo $SCRIPTPATH
+PARSL_ROOT=$(dirname $(dirname $(dirname $SCRIPTPATH)))
+PARSL_GITHASH=$(git rev-parse --short HEAD)
+
+CONDA_TARGET=parsl_$PARSL_GITHASH.py3.7
+export CONDA_TARGET
+
+if [[ "$CONDA_TARGET" == "$CONDA_DEFAULT_ENV" ]]
+then
+    echo "Conda target env $CONDA_TARGET loaded"
+    exit 0
+fi
+
+
+create_conda() {
+    pushd .
+    cd $PARSL_ROOT
+
+    if [[ "$(hostname)" =~ .*thetalogin.* ]]
+    then
+        echo "On theta"
```

Comment:
having this stuff hardcoded into an if-statement rather than in separate files makes me cry